dataflow/src/main/java/org/checkerframework/dataflow/livevariable/LiveVarStore.java

```diff
@@ -14,47 +14,47 @@
 import org.checkerframework.dataflow.expression.JavaExpression;
 import org.checkerframework.javacutil.BugInCF;
 
-import java.util.HashSet;
 import java.util.LinkedHashSet;
 import java.util.Set;
 import java.util.StringJoiner;
 
 /** A live variable store contains a set of live variables represented by nodes. */
 public class LiveVarStore implements Store<LiveVarStore> {
 
-    /** A set of live variable abstract values. */
-    private final Set<LiveVarValue> liveVarValueSet;
+    /** A set of live variable values. */
```

Comment:
A set of live variable nodes that store ....

Reply:
Done.